$MirOS: ports/games/bsdgames/patches/patch-trek_USD_doc_Makefile,v 1.1.7.1 2005/03/18 15:46:07 tg Exp $
--- trek/USD.doc/Makefile.orig	Thu Jan  1 00:00:00 1970
+++ trek/USD.doc/Makefile	Sat Nov 19 22:53:34 2005
@@ -5,10 +5,11 @@ DIR=	usd/31.trek
 SRCS=	trek.me
 MACROS=	-me
 
-paper.ps: ${SRCS}
-	${TBL} ${SRCS} | ${ROFF} > ${.TARGET}
-
-paper.txt: ${SRCS}
-	${TBL} ${SRCS} | ${ROFF} -Tascii > ${.TARGET}
-
-.include <bsd.doc.mk>
+.ifdef TRUEPREFIX
+# MirPorts  
+install:
+	${INSTALL} ${INSTALL_COPY} -o ${DOCOWN} -g ${DOCGRP} -m ${DOCMODE} \
+	    Makefile ${SRCS} ${DESTDIR}${DOCDIR}/${DIR}/
+.else
+.  include <bsd.doc.mk>
+.endif
